src/lib/components/SvgRailroadGraph.svelte

```diff
@@ -0,0 +1,88 @@
+<script lang="ts">
+    import { columnColor } from './columnColor'
+
+    export let parentData: Map<number, number[]>
+    export let tableBody: HTMLTableSectionElement
+    export let columnWidth = 20
+    $: halfColumnWidth = columnWidth / 2
+    $: linkRadius = halfColumnWidth
+    // TODO: Add <a href="..."> links to individual commits (once we're ready to render individual commit diffs)
+    export let dotRadius = 3
```

Comment:
stylistically, you might consider keeping your exports together so the "api" to the component is immediately visible.  Then I'd suggest an empty line before your reactive declarations, i.e., `$:`'s.  FWIW, seeing things like this is most logical imo:

```javascript
...
export ...
export...

let ...
let ...

$: ...
$: ...
```
...